### PR TITLE
Display speaker status

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

File: js/directives/speakerCard.js [hunk 1]
```diff
@@ -8,7 +8,13 @@ require('../filters/filters.js');
 			scope : {
 				speaker: "="
 			},
-			templateUrl: "speakerCard.html"
+			templateUrl: "speakerCard.html",
+            link: function(scope, elem, attrs) {
+                var statusClasses = ["unassigned", "assigned", "in-progress", "deferred", "topic-approved", "date-assigned"];
+                scope.statusToCssClass = function(status) {
+                    return statusClasses[status];
```
Comment (dwybourn) [inline js/directives/speakerCard.js]: At the moment the speaker service doesn't return a status field, so currently the speaker card footer (admin and last contacted) have a background and text colour of white, backing them invisible. 

I'd suggest we have a default colour to use if the service doesn't provide a status, or gives us one we don't recognize.

**Comment by lclarksl** [comment]
@dwybourn I've set it to return the unassigned class if it doesn't get a status code that it recognises. I'll add in a task to decide how the UI should deal with unrecognised statuses?

**Comment by jebbinSCL** [comment]
So I think there are a few things we'll eventually need to do around the statuses and corresponding css.
1. We'll need to have an integration test to make sure that each status has a match css. This doesn't even need to be through the UI, it can just be reading over a mapping file of some form.
2. An unassigned status should be expected, it should be the exception. Thus, throw an error to be logged (when we get to logging) and display the default colour so the user isn't to alarmed by it.
3. We have to decide what the user is allowed to do in this case. If we are assuming every user is meant to be an admin anyway, we can say that we allow them to reset the status manually (Or they might be able to do this anyway.
4. Another thing we could potentially do is to automatically reset the status the first in the pipeline if an unexpected one is received. Doing this alone probably wouldn't be to great, as it could lead to data loss. However, if we have set data on what is required to get to each pipeline stage, we could potentially use that to decide the pipeline stage

In summary, I think we want to:
1. Make sure we have an intergration test for this
2. Show a default colour, but also make it clear that something has gone wrong in a non annoying way. 
3. Throw an error in the background and log. 
4. Allow the user to fix if they have the permission by letting them choose a stage in the pipeline or find a way to automatically decide what stage of the pipeline they should be in (Auto decide might be overkill)

**Comment by jebbinSCL** [comment]
:shipit:

**Comment by dwybourn** [comment]
:shipit:
